cmd/swarm/fs_test.go

```diff
@@ -58,7 +58,7 @@ func TestCLISwarmFs(t *testing.T) {
 	handlingNode := cluster.Nodes[0]
 	mhash := doUploadEmptyDir(t, handlingNode)
 	log.Debug("Mounting first run...")
-	log.Debug(fmt.Sprintf("ipc path: %s", filepath.Join(handlingNode.Dir, handlingNode.IpcPath)))
+	log.Debug("ipc path", filepath.Join(handlingNode.Dir, handlingNode.IpcPath))
```

Comment:
please fix logs, the args need to be `"description", "attr1", value1, "attr2", value2, ....`